### DISCUSSION TITLE
Installation and general questions

### DISCUSSION THREAD

**Discussion by reindert-vetter** [reindert-vetter/api-version-control#5] [General]
Is the documentation not completely clear? Then ask your question here.

**Comment by motaevns** [comment]
Hi, 
I think you should describe how to write routs correctly:
For me it works in next way
`
        Route::prefix('api/{v}/public')
            ->middleware(['api',])
            ->group(base_path('routes/api/v2/public.php'));
`

**Reply by reindert-vetter** [reply]
Thanks for your feedback. I have now described it more clearly:
https://github.com/reindert-vetter/api-version-control/tree/master#install

**Comment by jadsalhani** [comment]
Hello
I would say the docs need a concrete example that would work if reproduced, i.e route setup + config file sample + output URL + output result
Im actually facing some difficulty setting things up on my laravel install, so that would greatly help have a reference
